### PR TITLE
Labcoat sprite adjustments, MD-specific labcoat added

### PR DESCRIPTION
**Pull request by Anvilman6**
Labcoats now have little icons on the back related to their source job/department, and Medical Doctor now has a job-specific labcoat instead of using the generic white labcoat. As is per usual, I'm not anywhere near a competent spriter, so if anyone has feedback or changes to put forward, I certainly welcome the input.

# General Documentation

### Intent of your Pull Request
Existing labcoat worn sprites now have small icons on the back to indicate the job / department that the coat is associated with, complimentary to the colored arm stripes. Medical Doctors now have a job-specific labcoat they spawn with (this is also integrated into various relevant sources, such as the medi-drobe and medical uniform crate)

### Why is this change good for the game?
More immediate visual clarity in back view, less confusion from differing jobs sharing the same plain white labcoats, more fancy looking outwear for Med-Sci.

NOTE: SEE SPRITES LINKED IN THE PR COMMENT DISCUSSION BELOW FOR MOST RECENT VERSIONS
Current sprites:
Medical
image
Chemistry
image
Chemistry edited 7-1-2021
image
Genetics
image
Virology
image
Virology edited 7-1-2021
image
Science
image
Science edited 7-1-2021
image

# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
Changes existing labcoat back view worn sprites only, should be no Wiki changes needed there. 
Adds MD-specific labcoat with worn and item states which will need accounting for - Medi-drobe vendor inventory will need refreshed to reflect the new MD labcoat item sprite.

# Changelog

:cl:  
rscadd: Adds Medical Doctor specific labcoat, spawns on MDs, found in medi-drobe and from other sources
imageadd: Medical Doctor specific labcoat worn and item states
imageadd: Little back icons added to existing labcoats worn states
/:cl:

### CODE REVIEW

**Comment by ghost** [comment]
The cross on the Medical labcoat is **very slightly** off-center and I can't stop thinking about it.

**Comment by Anvilman6** [comment]
Hi all, I've pushed some updated versions of the Chemistry, Virology, and Science labcoats today. 

I kept the old photos on the PR documentation for recordkeeping reasons, and so people can give opinions between the old and new versions, if they would like. Hope you like them!

**Review by nesisknesis — CHANGES_REQUESTED** [review]
The designs are pretty unremarkable, the virologist is confusing, and the scientist is uninspired. This leads me to say the changes seem un-necessary, as I am also of the opinion that there is no problem with their clarity. While the MD labcoat is passable, I am fairly certain they're the only job that uses the plain labcoat so that would effectively make the former labcoat a dead item.

I feel like a more consistent and uniform design that would improve upon the current is the one used in the object sprites, with the coloured shoulders, as icons can leave quite a bit to interpretation and are harder to keep to a good standard, as demonstrated by the scientist's 'S'.

**Comment by Anvilman6** [comment]
image
Documentation in the main PR body will be updated soon, as I had this finalized the moment I have to walk out the door

**Review by nesisknesis — APPROVED** [review]

**Review by adamsong — APPROVED** [review]
Looks good code wise
